config_files/survey_year_2019_config.yml

```diff
@@ -4,94 +4,81 @@
-
-
-  ##############################################################################
-  # Paths to stratification files #
-  #################################
-
-  # file that relates the stratification to the Haul
-  # The two stratification types are found in two sheets: "Base KS" and "INPFC"
-  strata_filename: Stratification/US_CAN strata 2019_final.xlsx
-  strata_sheetname: Base KS
-
-  # file that defines the geographic definition of strata
-  # The two stratification types are found in two sheets: "stratification1" and "INPFC"
-  geo_strata_filename: Stratification/Stratification_geographic_Lat_2019_final.xlsx
-  geo_strata_sheetname: stratification1
-
-  # file that provides NASC values, which do not include age1 values
-  nasc_no_age1_filename: Exports/US_CAN_detailsa_2019_table2y+_ALL_final - updated.xlsx
-  nasc_no_age1_sheetname: Sheet1
-
-  # file that provides NASC values, which include all ages
-  nasc_all_ages_filename: Exports/US_CAN_detailsa_2019_table1y+_ALL_final - updated.xlsx
-  nasc_all_ages_sheetname: Sheet1
-  ##############################################################################
-
-
-  ##############################################################################
-  # Paths to Kriging files #
-  ##########################
-
-  # Mesh filename representing the centroids of the Kriging grid
-  mesh_filename: Kriging_files/Kriging_grid_files/krig_grid2_5nm_cut_centroids_2013.xlsx
-  mesh_sheetname: krigedgrid2_5nm_forChu
-
-  # smoothed contour used to transform the mesh points
-  smoothed_contour_filename: Kriging_files/Kriging_grid_files/transformation_isobath_coordinates.xlsx
-  #  smoothed_contour_filename: Kriging_files/Kriging_grid_files/Smoothing_EasyKrig.xlsx
-  smoothed_contour_sheetname: Smoothing_EasyKrig
-
-  vario_krig_para_filename: Kriging_files/default_vario_krig_settings_2019_US_CAN.xlsx
-  ##############################################################################
-
-
-  ##############################################################################
-  # Path to mapping between hauls and transects #
-  ###############################################
-
-  # File names for the US section
-  filename_haul_to_transect_US: Biological/US/haul_to_transect_mapping_2019.xlsx
-  haul_to_transect_US_sheetname: Sheet1
-
-  # File names for the Canada section
-  filename_haul_to_transect_CAN: Biological/CAN/haul_to_transect_mapping_2019_CAN.xlsx
-  haul_to_transect_CAN_sheetname: Sheet1
-  ##############################################################################
+##############################################################################
+# Parameters
+
+survey_year: 2019            # survey year being considered
+species_id: 22500            # target species ID for the survey year
+CAN_haul_offset: 200         # The value to be added to the Canadian's haul number
+hemisphere: NW               # north/south and west/east hemispheres, used for the reports
```

Comment:
Do we need to retain this in the YAML? I guess I don't know how this is used.

Reply:
You're referring to `hemisphere`, right? I had done a quick search through the code to see if it was used at all, and it isn't. I've confirmed this just now. So, I agree, let's drop it.

Reply:
I've pushed a commit removing `hemisphere`. Assuming you were referring only to that parameter, I'll flag this as resolved.